diesel/src/macros/mod.rs

```diff
@@ -891,8 +891,8 @@ macro_rules! __diesel_table_query_source_impl {
 /// The generated `ON` clause will always join to the primary key of the parent
 /// table. This macro removes the need to call [`.on`] explicitly, you will
 /// still need to invoke [`allow_tables_to_appear_in_same_query!`] for these two tables to
-/// be able to use the resulting query, unless you are using `infer_schema!` or
-/// `diesel print-schema` which will generate it for you.
+/// be able to use the resulting query, unless you are using `diesel print-schema`
+/// which will generate it for you.
```

Comment:
It's not immediately apparent to me what this "it" refers to.

Reply:
I agree. And it's not even clear to me either.

Reply:
It refers to `allow_tables_to_appear_in_same_query!.

Reply:
@weiznich Should I rephrase it, or leave it as it is then?

Reply:
If you have an idea how to rephrase it, then yes. Otherwise it is not required for merging this pr.